src/internal/sagaHelpers/retry.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-constant-condition */
+import { call } from '../io'
+import { delay } from '../utils'
+
+export default function* retry(maxTries, delayMs, worker, ...args) {
```

Comment:
the problem with this implementation is that it is generator based and we cannot distribute generators yet in the source code, as old browsers/nodes dont support it, library should be pretty much ready for usage in any es5-compatible environment when downloaded from npm

we also cannot transpile generators ourselves as it would require opinionated runtime to be included in out bundle (most popular being `regenerator-runtime`) which tend to be quite big

for those reasons other helpers are created with `fsmIterator` internal helper - it's quite easy to use, once you know what it does - I have once described it [here](https://github.com/redux-saga/redux-saga/issues/570#issuecomment-256717377)

It doesn't yet support errors though, so it would have need to be adjusted so it could support `retry` helper

Reply:
Gotcha. I thought the final output was transpiled. Sure, I can use the that iterator.